application/lib/cart.dart

```diff
@@ -40,14 +40,14 @@ class _CartState extends State<Cart> {
   }
 
   void _sendToServer() async {
-    final Set<Atom> atoms = EditorDisposition.of(context).cart;
+    final Set<Atom?> atoms = EditorDisposition.of(context)!.cart;
```

Comment:
EditorDisposition.of should be made non-nullable if we assume it is

Reply:
done